### PR TITLE
Improve security

### PR DESCRIPTION
**Pull request by babygoat**

### CODE REVIEW

**Comment by babygoat** [comment]
@nickhsine @taylrj

File: doc/prime-donation.apib [hunk 1]
```diff
@@ -97,7 +96,7 @@ The states *id* and *order_number* are assigned by the TWReporter Go API at the
 
 ### Modify a Single Prime Donation [PATCH]
 + Parameters
-    + id (number) ... ID of the Prime Donation
+    + order (number) ... an order number of the Prime Donation
```
Comment (taylrj) [inline doc/prime-donation.apib]: `order (string)` ?

**Comment by nickhsine** [comment]
LGTM 👍

File: doc/prime-donation.apib (hunk 1 above)
Comment (babygoat) [reply doc/prime-donation.apib]: nice gotcha!

**Comment by babygoat** [comment]
updated!

**Pull request merged**